index.html

```diff
@@ -0,0 +1,14 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <meta charset="utf-8">
+    <title>garden-vue</title>
+    <link rel="stylesheet" href="/static/normalize.css">
+    <script src="https://code.jquery.com/jquery-3.1.1.min.js" integrity="sha256-hVVnYaiADRTO2PzUGmuLJr8BLUSjGIZsDYGmIJLv2b8=" crossorigin="anonymous"></script>
+    <link rel="stylesheet" href="/static/semantic.min.css">
```

Comment:
Per our previous style conversation/guidelines, *please* use `mdl-lite`. 

https://getmdl.io/started/index.html see the npm section.